src/Sarif.Multitool/Sarif.Multitool.csproj

```diff
@@ -32,7 +32,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), build.props))\build.props" />
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp3.0;netcoreapp2.1;net461</TargetFrameworks>
+    <TargetFrameworks>netcoreapp3.1;netcoreapp2.1;net461</TargetFrameworks>
```

Comment:
>netcoreapp3.1 [](start = 22, length = 13)

This will change the output directory, so just a reminder that the Transformers project will also require this change too.

Reply:
I will revert this as the check-in doesn't contain other required clean-up as well.